### ISSUE TITLE
Error when installing Godot code support

### ISSUE THREAD

**Issue by Mister-Muffin** [atom/atom#19836]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

I can not install the package lang-gdscript.

**Error message:**
```
> tree-sitter-gdscript@0.2.3 install /tmp/apm-install-dir-119725-15300-wyjp48.qkzu/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
> prebuild-install || node-gyp rebuild


prebuild-install WARN install No prebuilt binaries found (target=3.1.10 runtime=electron arch=x64 libc= platform=linux)
Aborted (core dumped)
Aborted (core dumped)
gyp: Call to 'node -e "require('nan')"' returned exit status 134 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Linux 5.0.0-25-generic
gyp ERR! command "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/apm-install-dir-119725-15300-wyjp48.qkzu/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
gyp ERR! node -v v10.2.1
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
Aborted (core dumped)
npm WARN enoent ENOENT: no such file or directory, open '/tmp/apm-install-dir-119725-15300-wyjp48.qkzu/package.json'
npm WARN apm-install-dir-119725-15300-wyjp48.qkzu No description
npm WARN apm-install-dir-119725-15300-wyjp48.qkzu No repository field.
npm WARN apm-install-dir-119725-15300-wyjp48.qkzu No README data
npm WARN apm-install-dir-119725-15300-wyjp48.qkzu No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 134
npm ERR! tree-sitter-gdscript@0.2.3 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 134
npm ERR! 
npm ERR! Failed at the tree-sitter-gdscript@0.2.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/<user>/.atom/.apm/_logs/2019-08-25T16_39_35_655Z-debug.log
```

### Steps to Reproduce:
- Clicked install in lang-gdscript extension.

**Expected behavior:**
Sucessful install.

**Actual behavior:**
Error (not installing).

**Comment by rsese** [comment]
Thanks for the report - I think this could be related to https://github.com/atom/atom/issues/19780.

Looks like the `Versions` information wasn't filled out, what version of Atom are you running (`atom -v`)?  And what operating system?

**Comment by MarkusMCPEGamer** [comment]
Okay, thank you dir your Response! Of course I'll catch up later!

**Comment by Aerijo** [comment]
>  I think this could be related to #19780 

I don't think so; the flags are all correct. The best solution is for the `tree-sitter-gdscript` package to upload prebuilt binaries though.

For the error,
> gyp: Call to 'node -e "require('nan')"' returned exit status 134 while in binding.gyp. while trying to load binding.gyp

I experienced this personally at one point. Eventually I discovered I had NodeJS installed through `apt` somehow. Deleting it and using `nvm` installed NodeJS fixed the issue. I still don't know why it was an issue though.
